core/controllers/library.py

```diff
@@ -74,16 +74,7 @@ class LibraryPage(base.BaseHandler):
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
-        search_mode = 'search' in self.request.url
```

Comment:
Why delete? What happens when you do a search in the library?

Reply:
This is just used for the meta tag and @vojtechjelinek said it was okay to use library meta instead